### ISSUE TITLE
Feature request: Mesh transform filter with CoreImage

### ISSUE THREAD

**Issue by K-Be** [YuAo/Vivid#1]
I've tried to implement this filter, but faced with calculation limit. Have you tried to develop mesh transform filter?

**Comment by YuAo** [comment]
Here's a great project by Ciechan that implements the mesh transform filter using OpenGLES, [BCMeshTransformView](https://github.com/Ciechan/BCMeshTransformView). I haven't tried to implement a mesh transform filter, I believe you can get some ideas from Ciechan's project.

**Comment by K-Be** [comment]
By using OpenGLES I've implemented it. It is  simple. But I want to transform image with a dimension >= 8000, and can't do it with OpenGLES. I've processed preview image in OpenGLES, but can't generate result image because faced with Calculation Limit. I want to use mesh with size 100 \* 100 vertexes (every rectangle is presented with 2 triangles). 
I've find liquify example http://www.vidvox.net/forums//viewtopic.php?f=25&t=37920, but can't solve how to adopt it for mesh. 
One of the ideas is make every triangle transform as one transform in accumulator.  Now I'm working on it. 

Sorry, English is not my native language.
